kotlin/src/main/java/app/rive/runtime/kotlin/RiveAnimationView.kt

```diff
@@ -316,38 +317,65 @@ class RiveAnimationView(context: Context, attrs: AttributeSet? = null) : View(co
     }
 
     /**
-     * Load the [rive file][File] into the view.
+     * Create a view file from a byte array and load it into the view
      *
-     * - Optionally provide an [artboardName] to use, this defaults to the first artboard in the file.
-     * - Optionally provide an [animationName] to load by default, playing without any suggested animations names will simply play the first animations.
+     * - Optionally provide an [artboardName] to use, or the first artboard in the file.
+     * - Optionally provide an [animationName] to load by default, playing without any suggested animations names will simply play the first animaiton
      * - Enable [autoplay] to start the animation without further prompts.
      * - Configure [alignment] to specify how the animation should be aligned to its container.
      * - Configure [fit] to specify how and if the animation should be resized to fit its container.
      * - Configure [loop] to configure if animations should loop, play once, or pingpong back and forth. Defaults to the setup in the rive file.
      *
      * @throws [RiveException] if [artboardName] or [animationName] are set and do not exist in the file.
      */
-    fun setRiveFile(
-        file: File,
-        artboardName: String? = drawable.artboardName,
-        animationName: String? = drawable.animationName,
+    fun setRiveBytes(
+        bytes: ByteArray,
+        artboardName: String? = null,
+        animationName: String? = null,
         autoplay: Boolean = drawable.autoplay,
-        fit: Fit = drawable.fit,
-        alignment: Alignment = drawable.alignment,
-        loop: Loop = drawable.loop,
+        fit: Fit = Fit.CONTAIN,
+        alignment: Alignment = Alignment.CENTER,
+        loop: Loop = Loop.NONE,
     ) {
```

Comment:
Not sure the way these initialize make sense; if we're setting the Rive file multiple times, it'll reuse the first one's artboard and animation names if they're not specified, which will probably cause new files to explode. Better to have those reset to null so defaults can be chosen?

Same wit the other parameters; best to have everything reset to defaults?